flocker/control/httpapi.py

```diff
@@ -74,6 +74,29 @@ def version(self):
         """
         return {u"flocker":  __version__}
 
+    @app.route("/datasets", methods=['GET'])
```

Comment:
I would suggest this and the `POST` get changed to `/configuration/datasets` as you suggested at some point, so they are more symmetric with `/state/datasets` and it's immediately clear one is referring to *configuration*, not state.

(Might want to change this again when we have API that can actually tell you when a change has been applied successfully, but that may be a while...)